scripts/quickstart_run.sh

```diff
@@ -105,17 +106,35 @@ echo "Pods ready. Initializing environment"
 set -e
 
 # setup postgres db
-# dont fail if alredy exists
+# don't fail if already exists
 set +e
 export POSTGRES_POD="$(kubectl get pods --no-headers -o custom-columns=":metadata.name" | grep postgres)"
 kubectl exec $POSTGRES_POD -- psql -lqt | cut -d \| -f 1 | grep cello
 if [ $? != 0 ]; then
   kubectl exec $POSTGRES_POD -- createdb cello -U postgres
-  kubectl cp ./scripts/createdbtables.sql $POSTGRES_POD:./createdbtables.sql
-  kubectl exec $POSTGRES_POD -- psql -U postgres -d cello -f ./createdbtables.sql
 fi
 set -e
 
+# run DB migration job after Postgres DB is fully stood up
+echo "Applying DB migration manifest"
+kubectl apply -f ./scripts/quickstart_db_migration_manifest.yaml
```

Comment:
Opted to create a separate Kubernetes config for the migrate-db job instead adding it into the existing `quickstart_manifest.yaml`. Doing so would create a race condition between the Postgres pod and the migrate-db job without adding some hacky `initContainers` to the migrate-db job spec to check if Postgres is fully stood up.